xyo-android-library/src/main/java/network/xyo/sdk/XyoNodeBuilder.kt

```diff
@@ -1,12 +1,15 @@
 package network.xyo.sdk
 import android.content.Context
-import java.lang.Exception
-import java.nio.ByteBuffer
+import android.content.Context.LOCATION_SERVICE
```

Comment:
I'm not sure if these imports are used. These are already used in the `MainActivity` of the sample app, which is where I believe you wanted to include GPS in bound witness??

Reply:
Yes. My mistake to include this on the PR. This was a test that i've done to include GPS in bound witness. But didn't work first.